payloads/extensions/get_os.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
The same functionality was added to the GET extension. If you remove this, we are good to merge